hvplot/interactive.py

```diff
@@ -0,0 +1,362 @@
+    def __radd__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.add, other, reverse=True)
+        return self._clone(transform)
+    def __rand__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.and_, other)
+        return self._clone(transform)
+    def __rdiv__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.div, other, reverse=True)
+        return self._clone(transform)
+    def __rfloordiv__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.floordiv, other, reverse=True)
+        return self._clone(transform)
+    def __rlshift__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.rlshift, other)
+        return self._clone(transform)
+    def __rmod__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.mod, other, reverse=True)
+        return self._clone(transform)
+    def __rmul__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.mul, other, reverse=True)
+        return self._clone(transform)
+    def __ror__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.or_, other, reverse=True)
+        return self._clone(transform)
+    def __rpow__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.pow, other, reverse=True)
+        return self._clone(transform)
+    def __rrshift__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.rrshift, other)
+        return self._clone(transform)
+    def __rsub__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.sub, other, reverse=True)
+        return self._clone(transform)
+    def __rtruediv__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.truediv, other, reverse=True)
+        return self._clone(transform)
+
+    def plot(self, *args, **kwargs):
```

Comment:
Can you add a docstring explaining why `plot` needed this special code?  Does it mean that `.interactive` won't work with other .plot-API packages like cufflinks/pdvega, and/or related packages like Plotly Express?

Reply:
I'll add a comment, and no. It's really just matplotlib which has this implicit state nonsense, which requires me to work around. I also still have to add special handling for Pandas plot engines.